packages/adapters-library/src/scripts/templates/blankAdapterTemplate.ts

```diff
@@ -39,35 +36,30 @@ export function blankAdapterTemplate({
       UnderlyingReward,
     } from '../../../../types/adapter'
     import { Erc20Metadata } from '../../../../types/erc20Metadata'
-    import { IProtocolAdapter } from '../../../../types/IProtocolAdapter'
+    import { IProtocolAdapter, ProtocolToken } from '../../../../types/IProtocolAdapter'
     import { Helpers } from '../../../../scripts/helpers'
     import { RewardsAdapter } from '../../../../scripts/rewardAdapter'
     import { NotImplementedError } from '../../../../core/errors/errors'
     import { Replacements } from '../../../../scripts/replacements'
-    
-    type Metadata = Record<
-      string,
-      {
-        protocolToken: Erc20Metadata
-        underlyingTokens: Erc20Metadata[]
-      }
-    >
-    
-    export class ADAPTER_CLASS_NAME implements IProtocolAdapter, IMetadataBuilder {
+
+    type AdditionalMetadata = { underlyingTokens: Erc20Metadata[] }
```

Comment:
I really think that `underlyingTokens` shouldn't `AdditionalMetadata`. I can't think of a single use case when we need metadata and this is not needed.

Reply:
as discussed needed while we have 2 versions of the metadata types